components/camera/client.go

```diff
@@ -50,11 +50,11 @@ func NewClientFromConn(ctx context.Context, conn rpc.ClientConn, name string, lo
 func (c *client) Read(ctx context.Context) (image.Image, func(), error) {
 	ctx, span := trace.StartSpan(ctx, "camera::client::Read")
 	defer span.End()
-	mimeType := gostream.MIMETypeHint(ctx, utils.MimeTypeRawRGBALazy)
+	mimeType := gostream.MIMETypeHint(ctx, utils.MimeTypeEmpty)
```

Comment:
this seems like we're setting nothing now. why not just pass it through?

Reply:
we determined that the server would determine the default MIME type if the client didn't set a hint or request one.

Reply:
got it. let's just use `""` then, not Empty

Reply:
https://docs.google.com/document/d/1kE-8PzFFvHrkxtsMpwrvoUiiAreWyPSUGMURcO6Ywa0/edit#